src/ui/MessageStatus/index.tsx

```diff
@@ -32,10 +32,10 @@ export default function MessageStatus({
   const status = useMemo(() => (
     getOutgoingMessageState(channel, message)
   ), [channel?.getUnreadMemberCount?.(message), channel?.getUndeliveredMemberCount?.(message)]);
-  const showMessageStatusIcon = channel?.isGroupChannel()
-    && !channel?.isSuper
-    && !channel?.isPublic
-    && !channel?.isBroadcast;
+  const hideMessageStatusIcon = channel?.isGroupChannel() && (
+    (channel.isSuper || channel.isPublic || channel.isBroadcast)
```

Comment:
is there chance of channel being empty?

Reply:
I guess no, if the channel is empty the `&&` operator won't run the latter part of condition